scripts/build.sh

```diff
@@ -1,5 +1,20 @@
 #!/bin/sh -e
 
+function log() {
```

Comment:
On my system, using Ubuntu through Windows subsystem for linux, `/bin/sh` by default points to dash, which does not have many modern scripting features, such as functions, so this script errors with `./scripts/build.sh: 3: Syntax error: "(" unexpected`. dash is the shell that `/bin/sh` points to by default on Ubuntu and other Debian-based linux distros, so many linux and windows (through WSL) users will run into this error, which is not trivial to debug. Using `/bin/bash` works for me. It would be good to either switch the shell utility used here or to add some detail to the README about this error